src/Symfony/Component/HttpKernel/Bundle/AbstractBundle.php

```diff
@@ -47,4 +47,16 @@ public function getContainerExtension(): ?ExtensionInterface
 
         return $this->extension ??= new BundleExtension($this, $this->extensionAlias);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPath(): string
+    {
+        if (null === $this->path && false !== strrpos($path = parent::getPath(), \DIRECTORY_SEPARATOR.'src')) {
```

Comment:
str_contains :)
no slash after src?
preg_match?

Reply:
> str_contains :)

do you mean `str_ends_with`?

> no slash after src?

at this point `$path` is the result of `dirname()` which returns a path without slash on the right.

> preg_match?

Idw, is it faster?